qemu/tests/qemu_guest_agent.py

```diff
@@ -1458,6 +1458,63 @@ def gagent_check_frozen_io(self, test, params, env):
 
         self.gagent_verify(self.params, self.vm)
 
+    @error_context.context_aware
+    def gagent_check_VSS_status(self, test, params, env):
+        """
+        Only for windows guest,check QEMU Guest Agent VSS Provider service start type
+        and if it works.
+
+        Steps:
+        1) Check VSS Provider service start type.
+        2) Check VSS Provider service should be in stopped status.
+        3) Issue fsfreeze qga command.
+        4) Check VSS Provider service should be in running status.
+        5) Issue fsthaw qga command.
+
+        :param test: kvm test object
+        :param params: Dictionary with the test parameters
+        :param env: Dictionary with test environment
+        """
+        def check_VSS_status(status_expect):
```

Comment:
Whether to use lowercase letters is more match with Python's naming rules? like check_vss_status()?

Reply:
Thanks @peixiu  update, pls have a check.
 (1/1) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.qemu_guest_agent.virtio_serial.gagent_VSS_status.q35: PASS (54.81 s)